### PR TITLE
Adds HMD Brightness controls to the FMS

### PR DESCRIPTION
**Pull request by FatLurch**
**When merged this pull request will:**
- Adds "BRT" and "DIM" controls to HMD FMS screen
- BRT and DIM adjust HMD brightness
- Default HMD color set to `[0,1,0]`
- Partially address #192 
- Delineate HMD brightness between left and right seats

https://youtu.be/YacpD5zkACI

### CODE REVIEW

**Review by YannikH — CHANGES_REQUESTED** [review]
Thsi will conflict with #221 as that takes dummy1 and makes it dummy11, coordinate with ampersand to make sure that this'll work after merged, I will approve it after

File: addons/uh60_anvishud/config/MFD/HMD.hpp
Comment (ampersand38) [inline addons/uh60_anvishud/config/MFD/HMD.hpp]: Why is the color changed? What does it look like now?

File: addons/uh60_anvishud/XEH_PREP.hpp
Comment (ampersand38) [inline addons/uh60_anvishud/XEH_PREP.hpp:4]: camelCase preferred.
hudDim
hudBrt

File: addons/uh60_fms/config/interaction.hpp
Comment (ampersand38) [inline addons/uh60_fms/config/interaction.hpp:194]: `buttonUp` code gets `_vehicle` passed through it. If you use `call fnc` then you already have it as a param.
see https://github.com/Project-Hatchet/H-60/blob/Main/addons/uh60_doorguns/config/cfgVehicles.hpp#L30
and https://github.com/Project-Hatchet/H-60/blob/Main/addons/uh60_doorguns/functions/fnc_coverSwitch.sqf

The 2 functions do the same thing, could consolidate to 
`[_vehicle, _delta] call vtx_uh60_anvishud_fnc_adjustBrightness` or similar, but not a big deal right now since we have to deal with colors.

File: addons/uh60_anvishud/functions/fnc_setup.sqf
Comment (ampersand38) [inline addons/uh60_anvishud/functions/fnc_setup.sqf]: Should be a CBA setting, although could wait for integrated setting via RGBA color.

File: addons/uh60_anvishud/functions/fnc_Hud_Brt.sqf
Comment (ampersand38) [inline addons/uh60_anvishud/functions/fnc_Hud_Brt.sqf]: if it's somehow 999 then it'll end up with 1099. Better off with something like 
`0 max _val min 1000`

**Comment by FatLurch** [comment]
> Thsi will conflict with #221 as that takes dummy1 and makes it dummy11, coordinate with ampersand to make sure that this'll work after merged, I will approve it after

I worked with ampersand - sounded like pylon indices up to dummy 7 were real and in use, the rest were provisional. I trimmed back to the ones that were in use and used 8 and 9 for the two different HMD brightnesses

**Review by YannikH — APPROVED** [review]
Looks good

**Review by ampersand38 — APPROVED** [review]

**Comment by ampersand38** [comment]
https://www.youtube.com/watch?v=YacpD5zkACI

**Comment by FatLurch** [comment]
https://youtu.be/YacpD5zkACI

**Pull request closed without merging**
